docs/modules/data-structures/pages/listening-for-map-entries.adoc

```diff
@@ -94,8 +105,24 @@ C++::
 --
 [source,cpp]
 ----
-
-
+#include "Employee.h"
```

Comment:
why don't we use include file statement similar to java version at line [89](https://github.com/hazelcast/hz-docs/pull/497/files#diff-3a0f447206aa7115457ee5e29aea9409826343642e7e1fec90ee94ae0af9554aL89) above.

Reply:
there is a ROOT folder to keep all the java example codes, there isn't for any other client. Maybe it can be restructured for other clients as well, but I think it should be discussed.